packages/jetpack-ai-calypso/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## v1.0.0 (2023-12-11)
```

Comment:
Might need to alter the date before merging.

Feel free to use an alpha version too, and omit the date, see the state-utils package for an example.